Entities/ACrab.cpp

```diff
@@ -2936,14 +2930,22 @@ void ACrab::DrawHUD(BITMAP *pTargetBitmap, const Vector &targetPos, int whichScr
         }
 
         // Weight and jetpack energy
-        if (m_pJetpack && m_pJetpack->IsAttached() && m_MoveState == JUMP)
-        {
-            float mass = GetMass();
-// TODO: Don't hardcode the mass indicator! Figure out how to calculate the jetpack threshold values
-            str[0] = mass < 135 ? -31 : (mass < 160 ? -30 : -29); str[1] = 0;
-            // Do the blinky blink
-            if ((str[0] == -29 || str[0] == -30) && m_IconBlinkTimer.AlternateSim(250))
-                str[0] = -28;
+        if (m_pJetpack && m_pJetpack->IsAttached() && m_Controller.IsState(BODY_JUMP))
```

Comment:
Because this is how the `AHuman` variant does this, and referring to MoveState makes the icon flash rapidly instead of properly displaying the "E for Empty" designator.